moveit_ros/occupancy_map_monitor/include/moveit/occupancy_map_monitor/occupancy_map_updater.h

```diff
@@ -66,6 +66,13 @@ class OccupancyMapUpdater
   /** \brief This is the first function to be called after construction */
   void setMonitor(OccupancyMapMonitor* monitor);
 
+  /** \brief Bypass the default monitor by setting the linked occupancy map directly.  Updates must be processed
+   * manually*/
+  void setTree(const OccMapTreePtr& tree)
+  {
+    tree_ = tree;
```

Comment:
Maybe consider wrapping this in the same update mutex, this could have strange effects when called during an update.